README.md

```diff
@@ -64,25 +66,27 @@ GitHub Action for automatically syncing LeetCode submissions to a GitHub reposit
 - `leetcode-csrf-token` _(required)_: The LeetCode CSRF token for retrieving submissions from LeetCode
 - `leetcode-session` _(required)_: The LeetCode session value for retrieving submissions from LeetCode
 - `filter-duplicate-secs`: Number of seconds after an accepted solution to ignore other accepted solutions for the same problem, default: 86400 (1 day)
-- `destination-folder` _(optional)_: The folder in your repo to save the submissions to (necessary for shared repos)
+- `destination-folder` _(optional)_: The folder in your repo to save the submissions to (necessary for shared repos), default: _none_
+- `verbose` _(optional)_: Requires an additional API call but adds submission percentiles and question numbers to the repo, default: true
```

Comment:
nit: Maybe put the description first, e.g. 

Adds submission percentiles and question numbers to the repo (requires an additional API call)